src/stories/customErrorPage.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from "react"
+
+import { StoryProps } from "./storyProps"
+import StoryContainer from "./storyContainer"
+import CustomErrorPage, { Error404Props } from "../components/customErrorPage"
+
+export default {
+  title: "Error page",
+  component: CustomErrorPage,
+  args: {
+    statusCode: 404,
+    image: "error404.png",
```

Comment:
while this might work in the context of storybook, I don't think it will in the project. we'd have to force the image through a require so it's picked up by webpack, no?

Reply:
yes. you have to write `require("~/assets/error404.png")`

should I pass the image as a function?

Reply:
oh, got it. no worries then. it will check out in the project as the require essentially also returns the resulting path as a string